projects/plugins/boost/wp-js-data-sync.php

```diff
@@ -260,8 +260,16 @@ function jetpack_boost_initialize_datasync() {
 					)
 				)
 			),
-			'startDate' => Schema::as_number(),
-			'endDate'   => Schema::as_number(),
+			'annotations' => Schema::as_array(
+				Schema::as_assoc_array(
+					array(
+						'timestamp' => Schema::as_number(),
+						'text'      => Schema::as_string(),
+					)
+				)
+			)->nullable(),
+			'startDate'   => Schema::as_number(),
+			'endDate'     => Schema::as_number(),
```

Comment:
Just making sure - is changing the schema with an already released feature not going to affect already existing data?

Reply:
The data is coming from WPCOM, not being stored in the plugin. So, there shouldn't be any problem like that.

Reply:
And it's not cached in a transient either?